### DISCUSSION TITLE
Copilot chat extension crashes and sidebar item disappear in VS Code

### DISCUSSION THREAD

**Discussion by andyjeffries** [community/community#113158] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I'm running VS Code on MacOS Sonoma 14.2.1

I have a paid Copilot subscription via my company.

The copilot chat icon appears when I launch VS Code, but disappears a few seconds after.

Screen recording:

https://github.com/community/community/assets/22904/292e070a-ce7f-48f7-8b07-1d0b56a2c559

The screen recording also shows that I am running the latest Copilot Chat extension.

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

1. Please check on GitHub in settings - Copilot if you don't have any pending items.

2. Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

<hr>

If I supported you consider to mark my answer

**Reply by andyjeffries** [reply]
I've marked it... I had no pending items on GitHub, so raised a ticket.

**Reply by andyjeffries** [reply]
I've unmarked it as the answer, because you pointed me to GitHub support and they provided the resolution - so I've pasted it as a separate answer and marked that (to help anyone else finding this thread in future)

**Comment by andyjeffries** [comment, marked as answer]
The answer was that the organisation didn't have Copilot Chat enabled. I don't know why it's not enabled by default OR that the extension doesn't output a message explaining before just disappearing, but enabling it works for me.

https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#enabling-or-disabling-github-copilot-chat-3
